RFC-0001-prsestissimo-metrics.md

```diff
@@ -0,0 +1,355 @@
+## **Prestissimo Runtime Metrics Capturing and reporting**
+
+
+## **Proposers**
+
+* Ahana, an IBM company.
+* Uber.
+
+## **Prototype PR**
+
+There are two approaches in this PR:[prestodb/presto#21599](https://github.com/prestodb/presto/pull/21599) which will be discussed in detail.
```

Comment:
Remove this section `Prototype PR`. We mention this below anyway.

Reply:
Done.